### PR TITLE
fix mathml namespace issue when reparenting problems-exercises in ap-physics

### PR DESCRIPTION
**Pull request by kswanson33**
openstax/cnx-recipes#4622

AAAAAAAAAAAAAH!!!!

### CODE REVIEW

File: lib/recipes/ap-physics-2e/locales/en.yml [hunk 1]
```diff
@@ -4,5 +4,5 @@ en:
     glossary: Glossary
     section-summary: Section Summary
     conceptual-questions: Conceptual Questions
-    problems-exercises: Problems & Exercises
+    problems-exercises: Problems &amp; Exercises
```
Comment (philschatz) [inline lib/recipes/ap-physics-2e/locales/en.yml:7]: Could something like this work in general? `my_string.encode(:xml => :text)`

[StackOverflow](https://stackoverflow.com/a/7573833)

Comment (kswanson33) [reply lib/recipes/ap-physics-2e/locales/en.yml:7]: totally - i think we have a couple of options here. my first thought was to test this via a CI script, so I opened this card: https://github.com/openstax/cnx-recipes/issues/4657 

i wonder if surrounding the value text with quotes would also work, though. setting that as a norm (and enforcing it with some kind of linter) would maybe be easier. any thoughts?

Comment (philschatz) [reply lib/recipes/ap-physics-2e/locales/en.yml:7]: I was trying to figure out what nokogiri is doing and ended up with [these specs](https://github.com/openstax/cookbook/compare/bye-namespace) to help figure out when the `m:` is being removed from a document.

It seems that if any content is invalid (even is nokogiri has the STRICT flag set) then nokogiri will try to clean it up

**Comment by kswanson33** [comment]
phil & i decided to fix future instances of this by updating nokogiri to v1.13+ ([issue](https://github.com/openstax/cnx-recipes/issues/4657))

**Review by wilkus27 — APPROVED** [review]

**Pull request merged**